.vscode/extensions.json

```diff
@@ -1,12 +1,31 @@
 {
-  "recommendations": [
-    "andrewleedham.vscode-css-modules",
-    "bungcip.better-toml",
-    "clinyong.vscode-css-modules",
-    "dbaeumer.vscode-eslint",
-    "mgmcdermott.vscode-language-babel",
-    "silvenon.mdx",
-    "xyc.vscode-mdx-preview"
-  ],
-  "unwantedRecommendations": []
-}
```

Comment:
- [ ]  Please restore the extensions.json file

Reply:
This is caused by vscode.